data/bad_domains.txt

```diff
@@ -0,0 +1,66 @@
+ahoo.com
+aol.co.ul
+binternet.com
+btinternet.clm
+btopnworld.com
+example.com
+gamail.com
+gamail.co.uk
+gamil.com
+gamil.co.uk
+g.mail.com
+gmai.com
+gmail
+gmail.c
+gmail.con
+gmail.om
+gotmail.co.uk
+googmail.com
+hays-travelco.uk
+hitmail.com
+homail.com
+hormail.com
+hotma.com
+hotmaail.co.uk
+hotmaail.com
+hotmai.co.uk
+hotmai.com
+hotmailco.uk
+hotmail.co
+hotmail.co.ik
+hotmail.co.uj
+hotmail.co.ul
+hotmail.com.uk
+hotmail.uk
+hotmail.xo.uk
+hotmaio.com
+hotmal.com
+hotmiail.co.uk
+hotmial.com
+hotmil.com
+hotmqil.co.uk
+hotmsil.com
+hotnail.co.uk
+hoymail.co.uk
+hptmail.com
+htomail.co.uk
+htmail.com
+i.cloud.com
+icould.com
+iloud.com
+live.cco.uk
+live.co.ul
+live.om
+luve.co.uk
+mtlworld.com
+ricketmail.com
+yahaoo.com
+yahoomail.com
+yaho.co.uk
+yaho.com
+yahoi.com
+yahooo.com
+yahooco.uk
+yhoo.com
+ucloud.com
+viginmedia.co.uk
```

Comment:
Are the pluses meant to be there?

Separately, I'm a bit sad about this whole file. Attempting to enumerate every typo just seems like an absurd thing to do. Do we really want or need this?